src/Polly.Shared/PolicyAsync.cs

```diff
@@ -62,24 +84,62 @@ public Task ExecuteAsync(Func<Task> action)
         /// <param name="action">The action to perform.</param>
         /// <returns>The value returned by the action</returns>
         [DebuggerStepThrough]
-        public async Task<TResult> ExecuteAsync<TResult>(Func<Task<TResult>> action)
+        public Task<TResult> ExecuteAsync<TResult>(Func<Task<TResult>> action)
+        {
+            return ExecuteAsync(false, action);
+        }
+
+        /// <summary>
+        ///     Executes the specified asynchronous action within the policy and returns the result.
+        /// </summary>
+        /// <typeparam name="TResult">The type of the result.</typeparam>
+        /// <param name="continueOnCapturedContext">Whether to continue on a captured synchronization context.</param>
+        /// <param name="action">The action to perform.</param>
+        /// <returns>The value returned by the action</returns>
+        [DebuggerStepThrough]
+        public async Task<TResult> ExecuteAsync<TResult>(bool continueOnCapturedContext, Func<Task<TResult>> action)
```

Comment:
Request move `bool continueOnCapturedContext` to be last parameter, in `ExecuteAsync()`